app/stac_api/models.py

```diff
@@ -510,7 +510,14 @@ def upload_asset_to_path_hook(instance, filename=None):
     Returns:
         Asset file path to use on S3
     '''
-    logger.debug('Start computing asset file %s multihash', filename)
+    logger.debug(
+        'Start computing asset file %s multihash (file size: %s MB)',
+        filename,
+        int(instance.file.size / 1024**2),
+        extra={
+            "collection": instance.item.collection, "item": instance.item, "asset": instance.name
```

Comment:
`instance.item.collection` return the object not the name, same for `instance.item` ! It need to be `instance.item.collection.name` and `instance.item.name`.

Reply:
:+1: